src/service/config-store.ts

```diff
@@ -0,0 +1,56 @@
+import { HelpConfig } from '../help-config';
+import { ADMIN_CONFIG as adminConfig } from './configs/admin';
+import { AWARD_MANAGEMENT_CONFIG } from './configs/award';
+import { BB_HELP_CONFIG as bbHelpConfig } from './configs/bbhelp';
+import { config as bulkimportConfig } from './configs/bulkimport';
+import { config as churchConfig } from './configs/church';
+import { config as eduConfig } from './configs/edu';
+import { config as faithConfig } from './configs/faith';
+import { config as feConfig } from './configs/fe';
+import { config as fndliteConfig } from './configs/fndlite';
+import { config as gmkConfig } from './configs/grantmaking';
+import { config as marketplaceConfig } from './configs/marketplace';
+import { config as rexConfig } from './configs/rex';
+import { config as smallGroupsConfig } from './configs/smallgroups';
+import { STEWARDSHIP_MANAGEMENT_CONFIG } from './configs/stewardship';
+import { config as tcsConfig } from './configs/tcs';
+
+export type VALID_CONFIG_ID = 'admin'
```

Comment:
Probably doesn't matter here, but should this list be in alphabetical order when we create something like this for a supportal? ... Also, I wonder if this would be more intuitive for the supportal if this list was a dropdown too. We could have 2 dropdowns, with this list of svcids first, then the dropdown for locales, and finally a load button. That might be more intuitive in the supportal experience where writers are probably focused on just one svcid (and its locales in the rare cases when we have locales). What do you think?

Reply:
on the order of _this particular list_, the order doesn't _technically_ matter. this is j/ a type to help the typescript compiler understand what a valid id is. it's not generating the list in this project's demo page. but your point is still valid. keeping lists/types like this in alphabetical order makes it easier to read. i'll reorder this type.

on the order of _the list in `template.html`_ that generates the dropdown, your point is extra valid. i'll put those in a sane order. and we should do the same on the future supportal.

on the two-dropdown approach to the future supportal, agreed. that's likely a better experience. i'll go ahead and use that strategy on this demo page so you can see how that would roughly look